### PR TITLE
tidb cloud: add compression formats

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

added compression formats.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by dsdashun — APPROVED** [review]

File: tidb-cloud/naming-conventions-for-data-import.md [hunk 1]
```diff
@@ -48,20 +48,23 @@ This section describes the naming conventions for data files. Depending on the t
...
-`${db_name}.${table_name}${suffix}.csv`
+`${db_name}.${table_name}${suffix}.csv.${compress}`
 
 `${suffix}` is optional and can be one of the following formats, where *`xxx`* can be any number:
 
 - *`.xxx`*, such as `.01`
 - *`._xxx_xxx_xxx`*, such as `._0_0_01`
 - *`_xxx_xxx_xxx`*, such as `_0_0_01`
 
+`${compress}` is the compression format. TiDB Cloud supports the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`.
```
Comment (Frank945946) [inline tidb-cloud/naming-conventions-for-data-import.md:59]:
```suggestion
`${compress}` is the compression format and it is optional. TiDB Cloud supports the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`.
```

Comment (Frank945946) [reply tidb-cloud/naming-conventions-for-data-import.md:59]: @dsdashun PTAL

File: tidb-cloud/naming-conventions-for-data-import.md [hunk 2]
```diff
@@ -90,12 +93,21 @@ For example:
...
-- `${db_name}.${table_name}${suffix}.sql` (`${suffix}` is optional)
+`${db_name}.${table_name}${suffix}.sql.${compress}`
+
+`${suffix}` is optional and can be one of the following formats, where *`xxx`* can be any number:
+
+- *`.xxx`*, such as `.01`
+- *`._xxx_xxx_xxx`*, such as `._0_0_01`
+- *`_xxx_xxx_xxx`*, such as `_0_0_01`
+
+`${compress}` is the compression format. TiDB Cloud supports the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`.
```
Comment (Frank945946) [inline tidb-cloud/naming-conventions-for-data-import.md:104]:
```suggestion
`${compress}` is the compression format and it is optional. TiDB Cloud supports the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`.
```

File: tidb-cloud/naming-conventions-for-data-import.md (hunk 1 above)
Comment (dsdashun) [reply tidb-cloud/naming-conventions-for-data-import.md:59]: > @dsdashun PTAL

LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**